js/model/dinnerModel.js

```diff
@@ -1,14 +1,28 @@
 // DinnerModel Object constructor
 var DinnerModel = function () {
+  var _this = this;
+  var observers = [];
 
-  // TODO Lab 1 implement the data structure that will hold number of guest
-  // and selected dishes for the dinner menu
+  this.addObserver = function(observer) {
```

Comment:
As we won't have hundreds of events or observers. I think it's fine for us to keep all observers in one List and differentiate different events by passing a String to describe it as you are doing now.